### ISSUE TITLE
Could not find yaml of coco and vqvae

### ISSUE THREAD

**Issue by XuanjiaZ** [623851394/LG-VQ-language-guided-codebook-learning#1]
Thanks for this outstanding work!

I am very interested in it, but it seems to lack the YAML files related to COCO train. Could you release this file?
In addition, the paper refers to the VQVAE experiments, but I couldn't find the YAML file for VQVAE. Are these files not released, or is there some other tips to conduct VQVAE experiments?
I am looking forward to some solutions!!

Thanks again!

**Comment by GuotaoLiang** [comment]
Hi～I'm sorry for the late reply. This is coco.yaml, which contains three key hyperparameters that need to be carefully adjusted: nceloss_weight, mskloss_weight, wrsloss_weight. About VQVAE: I provide the model code of VQGAN, as well as yaml and loss files, you can refer to them.



| |
hinleung
|
|
***@***.***
|


---- Replied Message ----
| From | ***@***.***> |
| Date | 02/26/2025 16:38 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [GuotaoLiang/LG-VQ-language-guided-codebook-learning] Could not find yaml of coco and vqvae (Issue #1) |

Thanks for this outstanding work!

I am very interested in it, but it seems to lack the YAML files related to COCO train. Could you release this file?
In addition, the paper refers to the VQVAE experiments, but I couldn't find the YAML file for VQVAE. Are these files not released, or is there some other tips to conduct VQVAE experiments?
I am looking forward to some solutions!!

Thanks again!

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

XuanjiaZ created an issue (GuotaoLiang/LG-VQ-language-guided-codebook-learning#1)

Thanks for this outstanding work!

I am very interested in it, but it seems to lack the YAML files related to COCO train. Could you release this file?
In addition, the paper refers to the VQVAE experiments, but I couldn't find the YAML file for VQVAE. Are these files not released, or is there some other tips to conduct VQVAE experiments?
I am looking forward to some solutions!!

Thanks again!
